app/core/application.coffee

```diff
@@ -51,7 +51,7 @@ Application = {
 #      document.location.href = '//lenovogamestate.com/login/'
 
     Router = require('core/Router')
-    @isProduction = -> document.location.href.search('https?://localhost') is -1
+    @isProduction = -> document.location.href.search('https?://localhost') is -1 and document.location.href.search('172.') is not 0
```

Comment:
This is a change from https://github.com/codecombat/codecombat/pull/5475 and would not be here when this is rebased on master and merged in. 

It is there to help with Windows development.